charts/zaakbrug/templates/configmap.yaml

```diff
@@ -63,6 +63,7 @@ data:
   {{- with .Values.zaakbrug.zgw }}
   zaakbrug.zgw.zaak-identificatie-template: {{ .zaakIdentificatieTemplate }}
   zaakbrug.zgw.document-identificatie-template: {{ .documentIdentificatieTemplate }}
+  zaakbrug.zgw.document-besluit-template: {{ .besluitIdentificatieTemplate }}
```

Comment:
~~Could this be renamed to zaakbrug.zgw.besluit-identificatie-template to be in line with the other names.~~

I see that is was renamed in DeploymentSpecifics.properties. Here the left side needs to exactly match the name used in DeploymentSpecifics.